### ISSUE TITLE
Working on this?

### ISSUE THREAD

**Issue by Joepetey** [StickToTheScript/SRL-RS3#1]
How can I help with this? I can't promise to spend a lot of time on this, but I would definitely like to help!

**Comment by StickToTheScript** [comment]
Absolutely! All help is greatly appreciated! I personally would love to fix all the currently existing broken functions. You can find a list of these functions [here](https://github.com/StickToTheScript/SRL-RS3#broken-functions).

In addition, there are also a [collection of functions that I have not had the chance to test](https://github.com/StickToTheScript/SRL-RS3#untested-functions) to determine if they are functional or not, so if you would like to look through those and attempt to fix those as you go, that'd be great too!
